projects.go

```diff
@@ -100,3 +100,33 @@ func (p *ProjectService) DeleteProject(id int) (*Project, error) {
 
 	return result, nil
 }
+
+// CancelUpdate cancel of awx projects update.
```

Comment:
Should `ProjectService` needs these apis? Please follow the official endpoint design.